### PR TITLE
Adds constants for each api version in kruize

### LINKED ISSUE #1019: Update the list recommendation version to latest as the API structure is changed
In Kruize the API structure of the List recommendation is changed to a new format which is a new version of the API response. Kruize has different set of api's so each endpoint could have specific version to indicate the user that in which version format he can expect the output.

### PR DESCRIPTION
**Pull request by bharathappali**
This PR changes the list recommendation Service Object to have specific version of API rather than having the kruize object version.

Fixes #1019

### CODE REVIEW

**Comment by dinogun** [comment]
@bharathappali Also the docs all need to be updated to indicate the version change

**Comment by dinogun** [comment]
@bharathappali please also take a look at the JSONs / csvs in https://github.com/kruize/kruize-demos/tree/main/remote_monitoring_demo

**Comment by bharathappali** [comment]
> @bharathappali please also take a look at the JSONs / csvs in https://github.com/kruize/kruize-demos/tree/main/remote_monitoring_demo

Will Raise the PR for the demo to be updated

**Comment by bharathappali** [comment]
> @bharathappali Also the docs all need to be updated to indicate the version change

Done.

File: tests/setup.log [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+- API forwarder fixed ip and port
+
+API server - http for UI and grpc for internal endpoint
+
+Every request from UI or curl will go to API server, this will figure out kruize url's and port and then talk via GRPC and respond to curl or UI via http
+
+[
+  {
+    "version": "1.0",
```
Comment (dinogun) [inline tests/setup.log:13]: This needs to be v2.0 ?

**Comment by bharathappali** [comment]
> @bharathappali please also take a look at the JSONs / csvs in https://github.com/kruize/kruize-demos/tree/main/remote_monitoring_demo

Raised a PR in Kruize-Demos repo https://github.com/kruize/kruize-demos/pull/67

File: tests/setup.log (hunk 1 above)
Comment (bharathappali) [reply tests/setup.log:13]: I have skipped it as it's a log file. Will change it as well.

Comment (bharathappali) [reply tests/setup.log]: Done.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**